sopel/modules/help.py

```diff
@@ -136,13 +134,17 @@ def post_to_termbin(msg):
     'termbin': post_to_termbin,
 }
 
+HELP_CACHE_NAMESPACE = 'help-setting-cache'  # Set top-level memory key name
+
 
 class HelpSection(StaticSection):
     """Configuration section for this module."""
     output = ChoiceAttribute('output',
                              list(PASTEBIN_PROVIDERS),
                              default='clbin')
     """The pastebin provider to use for help output."""
+    show_server_host = ValidatedAttribute('show_server_host', bool, default=True)
+    """Show the IRC server's hostname/IP in the first line of the help listing?"""
 
 
 def configure(config):
```

Comment:
Well, with now two PRs that each added a setting to a module that previously had none, we're missing a somewhat critical docstring hack that the other core modules have. This can go in a separate commit within this PR, or a new one—up to you, because I honestly don't care either way. :stuck_out_tongue:

```suggestion
def configure(config):
    """
    | name | example | purpose |
    | ---- | ------- | ------- |
    | output | clbin | The pastebin provider to use for help output |
    | show\\_server\\_host | True | Whether to show the IRC server's hostname/IP at the top of command listings |
    """
```

(If it's not obvious, this is a raw Markdown table. It gets dumped directly to [the configuration page](https://sopel.chat/usage/module-configuration/) on Sopel's website. Until we find a way to build that page using proper introspection—class attribute docstrings _aren't accessible anywhere_, thanks Python—this is how we have to do things so it can be auto-generated for each new release. :frowning_face:)

Reply:
In separate commit.